src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final String[] COLORS = {"Red", "Green", "Yellow", "white", "Black", "Blue"};
```

Comment:
The color 'white' should start with an uppercase letter for consistency with other color names in the array.